### PR TITLE
Add setting to disable message sending option for bot

### PR DESCRIPTION
**Pull request by reytum**

### CODE REVIEW

File: app/src/main/assets/applozic-settings.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
   "toolbarColor": "",
   "statusBarColor": "",
   "richMessageThemeColor": "",
+  "hideKeyboardForBot": false,
```
Comment (devashishmamgain) [inline app/src/main/assets/applozic-settings.json:88]: are we hiding only keyboard while continuing to show the textfield? or we are hiding the whole?

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: app/src/main/assets/applozic-settings.json (hunk 1 above)
Comment (reytum) [reply app/src/main/assets/applozic-settings.json:88]: Hiding the whole. The keyboard would hide automatically once the text field is hidden

Comment (reytum) [reply app/src/main/assets/applozic-settings.json:88]: Screenshot_20210304-212004

**Comment by devashishmamgain** [comment]
Any other generic name for setting which can be used in web also for this
feature?

**Comment by reytum** [comment]
How about disableCustomMessageForBots or disableSendMessageForBots ?

**Comment by devashishmamgain** [comment]
restrictMessageTyingWithBots ?

**Comment by reytum** [comment]
Looks good.

**Review by devashishmamgain — APPROVED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Pull request merged**